packages/core/admin/admin/src/content-manager/components/BlocksEditor/index.js

```diff
@@ -84,6 +109,13 @@ const BlocksEditor = React.forwardRef(({ intlLabel, name, readOnly, required, er
           </Typography>
         </Box>
       )}
+      {mediaLibVisible && (
+        <MediaLibraryDialog
+          allowedTypes={[ALLOWED_MEDIA_TYPE]}
+          onClose={handleToggleMediaLib}
+          onSelectAssets={handleSelectAssets}
+        />
+      )}
```

Comment:
That's interesting, do you see this not conflicting with css indexes? I guess not because the modal is a portal?? I think the alternative could be to move this into a context local to the component… But it depends how much you see all this stuff being called everywhere, if it’s only one place collocation is probably more ideal.